controllers/disruption_controller_test.go

```diff
@@ -539,30 +540,40 @@ var _ = Describe("Disruption Controller", func() {
 
 		It("should create a cloud disruption but apply a host disruption with the list of cloud managed service ip ranges", func(ctx SpecContext) {
```

Comment:
I think we'll want a test where we delete and re-create the targets, and then confirm the list of hosts didn't grow between injections. wdyt?

Reply:
we can add that as well yes.